tests/storages_tests/rdb_tests/create_db.py

```diff
@@ -24,6 +65,20 @@ def mo_objective_test_upgrade(trial: optuna.trial.Trial) -> Tuple[float, float]:
     return x, x**2 + y**2 + z**2
 
 
+def objective_test_upgrade_v3_distribution(trial: optuna.trial.Trial) -> float:
+    x1 = trial.suggest_float("x1", -5, 5)
+    x2 = trial.suggest_float("x2", 1e-5, 1e-3, log=True)
+    x3 = trial.suggest_float("x3", -6, 6, step=2)
+    y1 = trial.suggest_int("y1", 0, 10)
+    y2 = trial.suggest_int("y2", 1, 20, log=True)
+    y3 = trial.suggest_int("y3", 5, 15, step=3)
+    z = cast(float, trial.suggest_categorical("z", [-5, 0, 5]))
+    trial.set_system_attr("a", 0)
+    trial.set_user_attr("b", 1)
+    trial.report(0.5, step=0)
```

Comment:
We don't have set these attributes in this function judging from its use case.

Reply:
Oh, you're right... 01b9afb removes them. :bow: